lib/routes/note_page.dart

```diff
@@ -98,15 +100,8 @@ class _NotePageState extends State<NotePage> {
   }
 
   void generateId() async {
-    Note lastNote;
-    List<Note> notes = await helper.listNotes(ReturnMode.ALL);
-    notes.sort((a, b) => a.id.compareTo(b.id));
-
-    if (notes.isNotEmpty) {
-      lastNote = notes.last;
-    }
-
-    if (note.id == null) note = note.copyWith(id: (lastNote?.id ?? 0) + 1);
+    var uuid = Uuid();
```

Comment:
Might need to add this to locator

Reply:
Did in [newest commit](https://github.com/HrX03/PotatoNotes/pull/34/commits/c6a0c51e3f9efb3078be64c62a013a3719ec0a82)